docs/file.md

```diff
@@ -0,0 +1,88 @@
+Chunked Upload
```

Comment:
I think this document needs some improvement. I don't think a developer would have a very good idea of how to use chunked uploads after reading it. Perhaps we could have an end-to-end example or something.